src/screens/orders/view/Orders.js

```diff
@@ -43,7 +46,7 @@ export default function Orders({ navigation }) {
   const [status, setStatus] = useState("");
   const [errors, setErrors] = useState({});
   const { user } = useAuthContext();
-  const [flag, setFlag] = useState(false);
+ // const [flag, setFlag] = useState(false);
```

Comment:
Just remove if its not tied to some feature which is already developed, but is going to be released in future.